src/functionalTest/groovy/com/autonomousapps/jvm/projects/IntegrationTestProject.groovy

```diff
@@ -15,11 +15,11 @@ final class IntegrationTestProject extends AbstractProject {
 
   final GradleProject gradleProject
 
-  IntegrationTestProject() {
-    this.gradleProject = build()
+  IntegrationTestProject(boolean correctIntegrationTestDependencies) {
```

Comment:
is "correct" being used as a verb or an adjective here?

Reply:
_adjective_  change to `withCorrectIntegrationTestDependencies`